source/widgets/settings_window/general_tab.py

```diff
@@ -331,3 +362,54 @@ def purge_temp_now(self):
                 message="Failed to purge temp folder. Some files may be in use.",
                 icon=PopupIcon.WARNING,
             )
+
+    def register_with_winget(self):
+        success = register_with_winget(sys.executable, self.parent.parent.version)
```

Comment:
```suggestion
        success = register_with_winget(sys.executable, str(self.parent.version))
```
self.parent.parent.version is NoneType me. `self.parent` is BlenderLauncher, so `self.parent.version` is correct

Plus str(...) because register_with_winget expects a string and doesn't automatically cast it for me. Not having this causes a `Failed to register with WinGet: Could not convert the data to the specified type.`

Reply:
<img width="1007" height="138" alt="image" src="https://github.com/user-attachments/assets/db54de04-f6f9-459a-80d3-e5ee66e8b185" />
I guess Pylance doesn't like assigning `parent` to the same-named class variable. I skipped the warning without any more thought because of all the `self.parent` errors it gave me 😅

Reply:
Yeah that's been bothering me for so long lol... basically since the rebirth of this project